v21.2/known-limitations.md

```diff
@@ -159,50 +159,15 @@ UNION ALL SELECT * FROM t1 LEFT JOIN t2 ON st_contains(t1.geom, t2.geom) AND t2.
 
 [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/59649)
 
-## Unresolved limitations
-
-### `IMPORT` into a `REGIONAL BY ROW` table
-
-CockroachDB does not currently support [`IMPORT`s](import.html) into [`REGIONAL BY ROW`](set-locality.html#regional-by-row) tables that are part of [multi-region databases](multiregion-overview.html).
-
-[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/61133)
-
-To work around this limitation, you will need to take the following steps:
-
-1. In the source database, export the [`crdb_region` column](set-locality.html#crdb_region) separately when exporting your data.
-
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    EXPORT INTO CSV 'nodelocal://0/src_rbr' FROM SELECT crdb_region, i from src_rbr;
-    ~~~
+### Using `RESTORE` with multi-region table localities
```

Comment:
We should also add that restoring global/RBT tables into a non-MR DB  is not supported + tracking issue: backupccl: allow restore of GLOBAL/RBT tables in a non-MR database #71502

And in the Backup limitations page of MR -- we should add that: Table level backups are NOT supported + tracking issue: backupccl: support for multi-region table backups #71180